test/puppetlabs/puppetdb/config_test.clj

```diff
@@ -61,6 +62,78 @@
     (testing "should default to half the available CPUs, even on single core boxes"
       (is (= (with-ncores 1) 1)))))
 
+(deftest blacklist-configuration
+  (testing "blacklist-configuration"
+    (testing "convert-blacklist-config produces regex patterns and errors when patterns invalid"
```

Comment:
Not critical, but nesting `testing` forms just appends the strings, so this is going to be a pretty long description.  In any case, I'd be tempted to choose shorter names, and since I don't think there's much per-test overhead here, also tempted to just break these up into smaller deftest-chunks, with meaningful test names instead of the testing strings.  That way when something breaks, we can run smaller pieces, say: `(deftest blacklist-conversion-returns-patterns-when-appropriate ...)`, `(deftest black-conversion-returns-errors-for-bad-patterns ...)`, etc.